package.json

```diff
@@ -16,6 +16,7 @@
     "express": "^4.15.3",
     "github": "^9.2.0",
     "mz": "2.6.0",
```

Comment:
I removed the `mz` dependency in an earlier commit (no explicit explanation or reasoning was given for it). I dislike adding more dependencies than necessary, but I'd like to hear your explanation for adding the `mz` dependency as opposed to a devDependency, especially when the dependency has admitted to making [applications run slower](https://github.com/normalize/mz#will-this-make-my-app-faster).